schema.sql

```diff
@@ -0,0 +1,287 @@
+      'Rejected',
+      'Graylisted',
+      'Blacklisted'
+    ) NOT NULL,
+    submitted_at TIMESTAMP NOT NULL,
+    updated_at TIMESTAMP,
+    registrant_notes TEXT,
+    admin_notes TEXT,
+    tx_hash VARCHAR,
+    FOREIGN KEY (registrant_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  pot_factory (
+    id VARCHAR PRIMARY KEY,
+    owner_id VARCHAR NOT NULL,
+    deployed_at TIMESTAMP NOT NULL,
+    source_metadata VARCHAR NOT NULL,
+    protocol_fee_basis_points INT NOT NULL,
+    protocol_fee_recipient_account VARCHAR NOT NULL,
+    require_whitelist BOOLEAN NOT NULL,
+    FOREIGN KEY (owner_id) REFERENCES account (id),
+    FOREIGN KEY (protocol_fee_recipient_account) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  pot_factory_admin (
+    pot_factory_id INT NOT NULL,
+    admin_id VARCHAR NOT NULL,
+    PRIMARY KEY (pot_factory_id, admin_id),
+    FOREIGN KEY (pot_factory_id) REFERENCES pot_factory (id),
+    FOREIGN KEY (admin_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  pot_factory_whitelisted_deployer (
+    pot_factory_id INT NOT NULL,
+    whitelisted_deployer_id VARCHAR NOT NULL,
+    PRIMARY KEY (pot_factory_id, whitelisted_deployer_id),
+    FOREIGN KEY (pot_factory_id) REFERENCES pot_factory (id),
+    FOREIGN KEY (whitelisted_deployer_id) REFERENCES account (id)
+  );
+
+CREATE TABLE
+  pot (
+    id VARCHAR PRIMARY KEY,
+    pot_factory_id INT NOT NULL,
+    deployer_id VARCHAR NOT NULL,
+    deployed_at TIMESTAMP NOT NULL,
+    source_metadata VARCHAR NOT NULL,
+    owner_id VARCHAR NOT NULL,
+    chef_id VARCHAR NULL,
+    name TEXT NOT NULL,
+    description TEXT NOT NULL,
+    max_approved_applicants INT NOT NULL,
+    base_currency VARCHAR NULL,
+    application_start TIMESTAMP NOT NULL,
+    application_end TIMESTAMP NOT NULL,
+    matching_round_start TIMESTAMP NOT NULL,
+    matching_round_end TIMESTAMP NOT NULL,
+    registry_provider VARCHAR NULL,
+    min_matching_pool_donation_amount VARCHAR NOT NULL,
+    sybil_wrapper_provider VARCHAR NULL,
+    custom_sybil_checks VARCHAR NULL,
+    custom_min_threshold_score INT NULL,
+    referral_fee_matching_pool_basis_points INT NOT NULL,
+    referral_fee_public_round_basis_points INT NOT NULL,
+    chef_fee_basis_points INT NOT NULL,
+    total_matching_pool VARCHAR NOT NULL,
+    total_matching_pool_usd DECIMAL(10, 2) NULL,
+    matching_pool_balance VARCHAR NOT NULL,
+    matching_pool_donations_count INT NOT NULL,
+    total_public_donations VARCHAR NOT NULL,
+    total_public_donations_usd DECIMAL(10, 2) NULL,
+    public_donations_count INT NOT NULL,
+    cooldown_end TIMESTAMP NULL,
+    all_paid_out BOOLEAN NOT NULL,
+    protocol_config_provider VARCHAR NULL,
+    FOREIGN KEY (pot_factory_id) REFERENCES pot_factory (id),
+    FOREIGN KEY (deployer_id) REFERENCES account (id),
+    FOREIGN KEY (owner_id) REFERENCES account (id),
+    FOREIGN KEY (chef_id) REFERENCES account (id),
+    FOREIGN KEY (base_currency) REFERENCES account (id)
+  );
```

Comment:
add:

`account_id VARCHAR NOT NULL`
`FOREIGN KEY (account_id) REFERENCES account (id)`